### PR TITLE
PyUnit and Robot test for registration page

### LINKED ISSUE #75: Testing Registration
using [robot](http://robotframework.org) write a [Selenium](http://www.seleniumhq.org) test:
- Valid user registration 
- Non-valid user registration.

using [PyUnit](http://pyunit.sourceforge.net) write a unit test for:
- Registration view [function](https://github.com/ACME-OUI/acme-web-fe/blob/master/apps/web_fe/views.py#L198)

### PR DESCRIPTION
**Pull request by mattben**
adding my two test for sprint 2. fixes #75

### CODE REVIEW

File: apps/web_fe/views.py [hunk 1]
```diff
@@ -204,6 +204,7 @@ def register(request):
     if request.method == 'POST':
         user_form = UserCreationForm(data=request.POST)
         if user_form.is_valid():
+            print user_form
```
Comment (chaosphere2112) [inline apps/web_fe/views.py:194]: This line should probably be removed.

**Comment by chaosphere2112** [comment]
The whole cover subdirectory probably isn't supposed to be there; you should remove it and add it to `.gitignore`.

File: apps/web_fe/tests.py [hunk 2]
```diff
@@ -287,3 +287,14 @@ def test_user_login_success(self):
...
 
     # TODO: make this so it checks the destination
+
+
+class TestUserRegistration(unittest.TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_user_registartion_fail(self):
```
Comment (chaosphere2112) [inline apps/web_fe/tests.py:360]: *registration

File: userdata/testAPI.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+'''
+Created on Mar 16, 2015
+
+@author: raju332
+'''
+import VeloAPI
```
Comment (chaosphere2112) [inline userdata/testAPI.py:6]: What's up with this file? Is it a test?

Comment (sterlingbaldwin) [inline userdata/testAPI.py:6]: Its the test that Bibi gave us for the VeloAPI.py. The whole userdata file should be in gitignore.

Comment (chaosphere2112) [inline userdata/testAPI.py:6]: Is it getting run? I didn't see any mention of it in the travis log. If it isn't, can we package it up in a Nose test or something? If it's not getting run, it's not doing any good.

Comment (sterlingbaldwin) [inline userdata/testAPI.py:6]: Its not getting run. It was given to us just as an example of how to use the API, but at this point I dont think we need it anymore. The functionality that it uses is getting tested in the actual django tests.

Comment (chaosphere2112) [inline userdata/testAPI.py:6]: OK, then I vote we nix this file.

**Comment by sterlingbaldwin** [comment]
Lets add the userdata folder to .gitignore and then merge this.

**Comment by sterlingbaldwin** [comment]
Im going to merge this, but we should spend some effort to clean up the repo a bit. I know most of the messyness comes from me, so Im going to do a PR later that should be much cleaner.

**Comment by sterlingbaldwin** [comment]
wrong button! Now it wont let me merge :(

**Comment by mattben** [comment]
@chaosphere2112 my branch had fallen behind (no surprise). once this runs can you merge if it passes please.

**Comment by mattben** [comment]
@chaosphere2112 @sterlingbaldwin what does this mean https://travis-ci.org/ACME-OUI/acme-web-fe/builds/68855478 did I miss something import last week?

**Comment by chaosphere2112** [comment]
@mattben I upgraded us to Django 1.8, and they changed the name of the testing module you need to import. Check out [this document for more info](https://docs.djangoproject.com/en/1.8/topics/testing/overview/)

**Comment by sterlingbaldwin** [comment]
Instead of (...).unittest.TestCase its just django.test import TestCase

and then for the test classes its
`
class TestSomething(TestCase):
`

**Pull request closed without merging**